### DISCUSSION TITLE
Oqtane 2.0 errors moving the installed website from a PC to another with different DB connection sting

### DISCUSSION THREAD

**Discussion by Scippy** [oqtane/oqtane.framework#1053] [Q&A] [answered]
Hi, I've followed the instruction for installing on IIS on [Oqtane Blog](https://www.oqtane.org/Resources/Blog/PostId/542/installing-oqtane-on-iis) and all work fine on my local environment.
When I repeat the IIS configuration on my WindowsServer2012R2 with IIS 8.5 the Oqtane application return to me "HTTP Error 404. The requested resource is not found." But only moving the installation from another PC with different connection string, with a clean Oqtane installation all work fine.
Any help about?
Thanks

**Comment by leigh-pointer** [comment]
Hi, If I were you just double check your IIS settings in Programs and Features [step 1].

**Comment by Scippy** [comment, marked as answer]
I've found the first problem, working on host file on IP 127.0.0.1 I had mistakenly added the binding on the server's public ip.
Now I receive an "HTTP Error 500.30 - ASP.NET Core app failed to start"

**Reply by Scippy** [reply]
I've checked with an clean install and it work fine. The already installed Oqtane website moved from my local envirnment to server give me the HTTP Error 500.30

**Reply by leigh-pointer** [reply]
Try running it from a command window.  In your root folder (not wwwroot but the one above) run Oqtane.Server.exe then browse to localhost:5001 to check that indeed the app runs.

**Reply by Scippy** [reply]
Thanks, I found the issue in appsettings.json, I've modified the Defaultconnection and missed double \\ after server name.
Now I've fixed it but something in connection strings give me still an issue.
My connection string is:
`"Data Source=192.168.10.21\\SQL2014EXPR;Initial Catalog=Oqtane;User ID=oqtane;Password=12345;"`
Executing Oqtane.Server.exe now I receive an error:
![Screenshot_1](https://user-images.githubusercontent.com/16607338/104310434-fe869e80-54d3-11eb-9253-6e8d5f8d564f.jpg)
With the original connetion string for localhost envirment it execute without errors.

**Reply by leigh-pointer** [reply]
Can you check that the Tenant table [BConnectionString on the Master record is the same as the connection appsettings.json

**Reply by Scippy** [reply]
Oh YESS! I've modified the connectionstring also in Tenant table and now it work fine.
Thank You!

**Reply by leigh-pointer** [reply]
Good news, pleased that I could help.
